### PR TITLE
fix: clean up support and faq

### PR DESCRIPTION
**Pull request by stevanlohja**
Ref: ETCM-3853

### CODE REVIEW

**Review by ShariffL — APPROVED** [review]

File: content/08-cardano-sidechains/03-example-evm-sidechain/09-discussion-faq.mdx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+This page answers some frequent questions for the Cardano Sidechains Toolkit and the proof of concept EVM sidechain application.
+
+## The Cardano Sidechains Toolkit
+
+These questions relate to the components IOG provides for building custom sidechains on Cardano.
+
+### Is the Sidechain Toolkit an open-source project?
+
+Yes, it will be open-sourced under the Input Output Global GitHub repository.
```
Comment (NeilBurgess42) [inline content/08-cardano-sidechains/03-example-evm-sidechain/09-discussion-faq.mdx:14]: Should be present tense, and should give the name of the repo (when we know it)

Comment (ShariffL) [reply content/08-cardano-sidechains/03-example-evm-sidechain/09-discussion-faq.mdx:14]: New URL repo: https://github.com/input-output-hk/sc-evm @NeilBurgess42 @stevanlohja

Comment (stevanlohja) [reply content/08-cardano-sidechains/03-example-evm-sidechain/09-discussion-faq.mdx]: Committed suggested change. Please review and resolve this conversation if okay.

**Review by ShariffL — APPROVED** [review]

**Review by ShariffL — APPROVED** [review]

**Review by drospa — APPROVED** [review]
approved, but cannot be merged yet

**Comment by ShariffL** [comment]
@martinseanhunt can you approve? Someone with writer access needs to approve changes for Cardano Docs

**Comment by stevanlohja** [comment]
The suggested changes in this PR have been applied to https://github.com/input-output-hk/cardano-documentation/commit/6b43c533724a04310e8341ced51cb265a1b51905 which makes this PR redundant. Closing it.

**Pull request closed without merging**